ehitchhiking-ui/src/app/shared/components/active-trip/active-trip.component.ts

```diff
@@ -1,17 +1,43 @@
 import {Component, Input, OnInit} from '@angular/core';
 import {ActiveTrip} from './active-trip';
 import {UserState} from '@shared/enums/UserState';
+import {DEFAULT_MAT_DIALOG_CLASS, MAT_DIALOG_WIDTH_MD, MAT_DIALOG_WIDTH_SM} from '@shared/constants/modal-constants';
+import {MatDialog} from '@angular/material';
+import {ConfirmationModalComponent} from '@shared/modals/confirmation-modal/confirmation-modal.component';
+import {ActiveTripsModalService} from '@shared/components/active-trips-modal/active-trips-modal.service';
 
 @Component({
 	selector: 'app-active-trip',
 	templateUrl: './active-trip.component.html',
 	styleUrls: ['./active-trip.component.sass'],
 })
 export class ActiveTripComponent implements OnInit {
-	constructor() {}
+	constructor(public dialog: MatDialog, public tripService: ActiveTripsModalService) {}
 	userState = UserState;
 
 	ngOnInit() {}
 
 	@Input() trip: ActiveTrip;
+	static readonly REMOVAL_CONFIRMATION_MESSAGE: string = 'Do you really want to remove this trip from the active?';
```

Comment:
Do you really want to decline this trip?

Reply:
changed